pxr/usdImaging/usdImagingGL/CMakeLists.txt

```diff
@@ -142,3 +146,4432 @@ pxr_build_test(testUsdImagingGLSurfaceShader
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLFaceVarying_Adaptive_Medium.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLFaceVarying
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+
+pxr_register_test(testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Low
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -complexity 1.1 -offscreen -lighting -camlight -shading wireOnSurface -frameAll -stage testUsdImagingGLFaceVaryingLinearInterp.usda -write testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Low.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Low.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLFaceVarying
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+
+pxr_register_test(testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Medium
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -complexity 1.2 -offscreen -lighting -camlight -shading wireOnSurface -frameAll -stage testUsdImagingGLFaceVaryingLinearInterp.usda -write testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Medium.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLFaceVaryingLinearInterp_Adaptive_Medium.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLFaceVarying
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+
+pxr_register_test(testUsdImagingGLFaceVaryingLeftHanded_Adaptive_Low
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -complexity 1.1 -offscreen -lighting -camlight -shading wireOnSurface -frameAll -stage testUsdImagingGLFaceVaryingLeftHanded.usda -write testUsdImagingGLFaceVaryingLeftHanded_Adaptive_Low.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLFaceVaryingLeftHanded_Adaptive_Low.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLFaceVarying
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+pxr_register_test(testUsdImagingGLShadingPrimvars
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -shading smooth -frameAll -stage testPrimvarNameHandlingInGL.usda -write testUsdImagingGLShadingPrimvars.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLShadingPrimvars.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLShadingPrimvars
+    ENV
+        HD_ENABLE_OPENSUBDIV3_ADAPTIVE=1
+)
+
+# The following tests are disabled since the stages are unable to load any .tx files
+# In theory these image files should be able to load through the hioOiio plugin but I have not
+# been able to get that too work. I did try to convert these images through OIIO's iconvert tool
+# but it also fails trying to open the .tx files
```

Comment:
The comment pretty much describes the issue I ran into with .tx files. I just want to point this out so we can discuss how we should handle these tests